code/ecdsap256/Hacl.P256.fsti

```diff
@@ -452,6 +458,9 @@ val ecp256dh_i:
 This function takes a 32-byte secret key, another party's 64-byte raw public
 key, and computeds the 64-byte ECDH shared key.
 
+The user MUST validate keys with `is_more_than_zero_less_than_order` and
```

Comment:
we could simply expose one function called `is_public_key_valid` :)

Reply:
thanks Marina, I've pushed further changes that update the names in the API to be more informative -- I have essentially chosen @victor-dumitrescu's naming conventions